cmd/ocm/cluster/create/create.go

```diff
@@ -106,6 +127,20 @@ func run(cmd *cobra.Command, argv []string) error {
 	}
 	clusterName := argv[0]
 
+	// Parse the expiration options
+	if len(args.expirationTime) > 0 {
+		t, err := parseRFC3339(args.expirationTime)
+		if err != nil {
+			return fmt.Errorf("unable to parse expiration time: %s", err)
+		}
+
+		expiration = t
+	}
+	if args.expirationSeconds != 0 {
```

Comment:
Wondering if we could use `time.ParseDuration()`

Reply:
@vkareh ParseDuration takes a string, but the duration is already being parsed by https://pkg.go.dev/github.com/spf13/pflag?tab=doc#FlagSet.DurationVar, so at this point in the code it's a time.Duration already

Reply:
of course, thanks!